README.md

```diff
@@ -132,9 +134,9 @@ The following are the default input options.
 }
 ```
 
-### `callback(result, options)`
+### `Promise.resolve(result,options)` / `Promise.reject(errors) / `callback(result, options)`
```

Comment:
I think you need another backpack after `Promise.reject(errors)`.

Reply:
Thanks for catching that.